index.html

```diff
@@ -0,0 +1,160 @@
+        </div>
+        <div class="menu mobile-only hidden" id="mobileMenu">
+          <div class="close_btn">
+            <img src="assets/images/icon-cancel.svg" alt="menu cancel" />
+            <div class="top-bar">
+              <div class="top-bar-left">
+                <ul class="mobile-only">
+                  <li><a href="index.html">Language</a></li>
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#">Events</a></li>
+                  <li><a href="#">Speakers</a></li>
+                  <li><a href="#">Contact</a></li>
+                </ul>
+              </div>
+          </div>
+         </div> 
+        </div> 
+      </nav>
+      <nav class="desktop-only mobile-none">
+        <div class="top-bar">
+          <ul class="sm-menu">
+            <li class="small-menu"><img src="assets/images/facebook.png" alt="logo facebook"></li>
+            <li class="small-menu"><img src="assets/images/twitter.png" alt="logo twitter"></li>
+            <li class="small-menu">English</li>
+            <li class="small-menu">My Page</li>
+            <li class="small-menu">Logout</li>
+          </ul>
+        </div>
+        <div class="bottom-bar">
+          <img src="assets/images/advanced-energy-vector-logo.png" alt="logo" />
+          <ul class="big-menu">
+          <!-- </ul> -->
+          <!-- <ul class="big-menu-right"> -->
+            <li class="big-menu-right">
+              <a href="index.html">Index</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="about.html">About</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="#program">Program</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="#speak">Speakers</a>
+            </li>
+            <li class="big-menu-right">
+              <a href="#sponsors">Sponsors</a>
+            </li>
+        
+            <li class="big-menu-right"><button class="contact">Contact Us</button>
+            </li>
+          </ul>
+        </div>
+        </nav>
+    <!-- Top Menu Desktop-->
+    </header>
+    <!-- Main Welcome Text-->
+    <section>
+      <div>
+        <span id="greeting" class="greeting">"Hello! Energy Event!</span>
+        <h1 class="event-name">ENERGY EVENT<br>
+          GLOBALLY<br>
+          2024</h1>
+        <p class="event-description">
+          World event about Renewable Energy, Solar Energy, Clean Energy, with fantastic speakers and workshops. Many
+          new
+          technologies are coming to the world, and we are going to discuss how to use them. Best of all, we are going
+          to
+          be the best event in the world on these technologies.
+        </p>
+      </div>
+      <div class="date-event">
+        <h2 class="date">2024.07.12(FRI) ~ 13(SAT)</h2>
+        <p class="place">
+          @ Oscar Niemeyer Hall, Conference room, Goiânia<br>
+        </p>
+      </div>
```

Comment:
- [ ] __[OPTIONAL]__ The font-size here is too big

Yours | Original
---- | ----
![image](https://user-images.githubusercontent.com/16058470/143623306-779bf949-3558-45b4-90a0-b2a0ef0a2e91.png) | ![image](https://user-images.githubusercontent.com/16058470/143623357-9a3687c7-4512-4120-9cf6-437f60097ac8.png)